toolkit/tools/pkg/rpmssnapshot/rpmssnapshot.go

```diff
@@ -53,42 +52,37 @@ const (
 	rpmSpecBuiltRPMRegexDistributionIndex
 	rpmSpecBuiltRPMRegexArchitectureIndex
 	rpmSpecBuiltRPMRegexMatchesCount
+
+	chrootName = "rpmssnapshot_chroot"
```

Comment:
This one may also be made function-local.

Btw. the constants above are global only for the sake of keeping them closer to the regular expression, so it's easier to see which capture groups the indeces refer to. I keep going back and forth on what's the best place to keep such consts.